package.json

```diff
@@ -16,6 +16,9 @@
     "bin",
     "lib"
   ],
+  "engines": {
+    "node": ">=10.13.0"
+  },
```

Comment:
Not sure about this, but since 8 is becoming EOL, I don't see the harm.

Reply:
Last time we had issues with node v6 installations on CI and I still don't see any reason to restrict CI setups because of this developer's library. I'd say we should be using please-upgrade-node as we didi before.

Reply:
Sure, I'll just drop this commit: https://github.com/okonet/lint-staged/pull/747/commits/43190cbcad7e2b85f7f463208ff5031059b96341